src/webgl/p5.Matrix.js

```diff
@@ -505,22 +514,22 @@ p5.Matrix.prototype.scale = function() {
  * @chainable
  * inspired by Toji's gl-matrix lib, mat4 rotation
  */
-p5.Matrix.prototype.rotate = function(a, axis) {
-  var x, y, z, _a, len;
+p5.Matrix.prototype.rotate = function(a, x, y, z) {
+  var _a, len;
 
   if (this.p5 && this.p5._angleMode === constants.DEGREES) {
     _a = polarGeometry.degreesToRadians(a);
   } else {
     _a = a;
   }
-  if (axis instanceof p5.Vector) {
-    x = axis.x;
-    y = axis.y;
-    z = axis.z;
-  } else if (axis instanceof Array) {
-    x = axis[0];
-    y = axis[1];
-    z = axis[2];
+  if (x instanceof p5.Vector) {
```

Comment:
same suggestion as above; add a note here that arguments array is not used for optimization purposes